holoviews/plotting/bokeh/callbacks.py

```diff
@@ -129,35 +178,42 @@ class Callback(object):
           return
         }}
 
+        // Initialize event queue and timeouts for Comm
         var comm_state = HoloViewsWidget.comm_state["{comm_id}"];
         if (comm_state === undefined) {{
-            comm_state = {{event: undefined, blocked: false, timeout: Date.now()}}
+            comm_state = {{event_buffer: [], blocked: false, time: Date.now()}}
             HoloViewsWidget.comm_state["{comm_id}"] = comm_state
         }}
 
-        function trigger() {{
-            if (comm_state.event != undefined) {{
-               var comm_id = comm_state.event["comm_id"]
-               var comm = HoloViewsWidget.comms[comm_id];
-               comm.send(comm_state.event);
-            }}
-            comm_state.event = undefined;
-        }}
-
-        timeout = comm_state.timeout + {timeout};
+        // Add current event to queue and process queue if not blocked
+        event_name = cb_obj.event ? cb_obj.event.event_name : undefined
```

Comment:
I think this line is expected to change - I've been requested to change how this works in Bokeh.